src/category_theory/sites/cover_lifting.lean

```diff
@@ -22,11 +23,15 @@ but they are actually equivalent via `category_theory.grothendieck_topology.supe
 ## Main definitions
 
 * `category_theory.sites.cover_lifting`: a functor between sites is cover_lifting if it
-pulls back covering sieves to covering sieves
+  pulls back covering sieves to covering sieves
+* `category_theory.sites.copullback`: A cover-lifting functor `G : (C, J) ⥤ (D, K)` induces a
+  morphism of sheaves in the same direction as the functor.
```

Comment:
It's not a morphism of sheaves, but a functor between categories of sheaves, right?